databricks/koalas/indexes.py

```diff
@@ -807,6 +807,25 @@ def name(self) -> str:
     def name(self, name: str) -> None:
         raise PandasNotImplementedError(class_name='pd.MultiIndex', property_name='name')
 
+    @property
+    def nlevels(self) -> int:
```

Comment:
i think we already implemented at #945 ?

now `MultiIndex` is using `Index`'s `nlevels` directly, since `MultiIndex` inherits `Index` and they have exactly same logic.